src/Core/Framework/Plugin/Command/MakerCommand.php

```diff
@@ -100,19 +100,48 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 $directory
             );
 
+            if (\method_exists($this->generator, 'disableAskConfirmQuestion')) {
```

Comment:
What is this change doing? What are you trying to fix here? We typically don't like to manipulate state like this so it would be good to understand what is happening here.

Reply:
Yes. This I add disableAskConfirmQuestion method due to disable $io->confirm(self::CLI_QUESTION). On creation plugin is usefull but not in MakerCommand because if a user wants to create something specific, the question is no longer needed. Have you any suggest to do better? I should add  disableAskConfirmQuestion method on GeneratorInterface?

Reply:
maybe we can add a prop to `PluginScaffoldConfiguration` ? like `shouldAskQuestions`. Then in `\Shopware\Core\Framework\Plugin\Command\MakerCommand` we set it to false. Then the generators can just query the config?  cc @jozsefdamokos maybe you have some opinion :)